### PR TITLE
Firmware flasher fix for the release notes show

### PR DESCRIPTION
**Pull request by limonspb**
Release notes for 4.2/4.1 etc are not shown now. Causing console error also:

image

This is a fix.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]
Supersedes https://github.com/betaflight/betaflight-configurator/pull/2824

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
@limonspb did we just use marked for firmware flasher tab?

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by limonspb** [comment]
> @limonspb did we only use marked for firmware flasher tab?

it was there for a long time. But something has changed in the marked library, someone updated it or something.

**Pull request merged**
